edgarduran/ideabox-2.0
# extracted html elements to another file

@jwperry - Let me know what you think about the refactoring and if you suggest anything different
@rrgayhart - Refactoring and tractoring
#### What's this PR do?

 This PR removes the html elements from the functions that take care of logic and performing jQuery actions and moves them into their own file. This is to achieve better code readability. 
#### Where should the reviewer start?

[New file](https://github.com/edgarduran/ideabox-2.0/blob/refactor-tractor/app/assets/javascripts/html-elements.js)
[Cleaned up file 1](https://github.com/edgarduran/ideabox-2.0/blob/refactor-tractor/app/assets/javascripts/ideas.js)
[Cleaned up file 2](https://github.com/edgarduran/ideabox-2.0/blob/refactor-tractor/app/assets/javascripts/ideaEdit.js)
[Look here for changes](https://github.com/edgarduran/ideabox-2.0/pull/9/files)
#### How should this be manually tested?

To test this you may want to clone that app and make sure ideas are still displaying and that you can create and edit. Or you can take my work for it that it works. 
#### What gif best describes this PR or how it makes you feel?

![](http://topfinancialadvisor.org/wp-content/uploads/2014/07/money-gifs-zombieland.gif)
